frontend/app/components/delete-with-confirmation.js

```diff
@@ -14,22 +14,38 @@ export default class DeleteWithConfirmation extends Component {
 
   @action
   openConfirmation() {
+    if (!this.get("router.currentURL").includes("skills")) {
```

Comment:
Not really happy with this solution.
Just setting the style value seems very random from an outstander / debugging perspective.
I think it would be much better to conditionally add a CSS class to the element.